matchmaker/views/matchmaker_api.py

```diff
@@ -50,44 +48,30 @@ def get_individual_mme_matches(request, submission_guid):
 
     results = MatchmakerResult.objects.filter(submission=submission)
 
-    response_json = get_json_for_saved_variants_with_tags(
```

Comment:
we were returning all the saved variants and their tags etc to show on the matchmaker summary page, but this is actually unnecessary as all we need for that page are the saved variants associated with the submission. All the saved variants are needed if the user is updating a submission, but opening the modal to do that editing alrady triggers a load of all the needed data, so returning the data here as well as redundant